docs/source/release/v6.8.0/Framework/Algorithms/Bugfixes/35787.rst

```diff
@@ -1 +1 @@
-- Fix for the :ref:`LoadEMU <algm-LoadEMU>` and :ref:`LoadPLN <algm-LoadPLN>` loader to capture all the environment parameters in the datafile.
\ No newline at end of file
+* Fix for the :ref:`LoadEMU <algm-LoadEMU>` and :ref:`LoadPLN <algm-LoadPLN>` loader to capture all the environment parameters in the datafile.
```

Comment:
I always come up against this when doing the release editor stuff, we should probably standardise either `-` or `*` because it's likely to be very confusing for someone if their PR is failing because of it.

Reply:
I agree, I think we need to add some documentation on this. Having this knowledge spread wider across the team will also help

Reply:
Is there any existing documentation on release notes? I can add it to that documentation in a separate PR

Reply:
It's actually already listed on line 47 of `ReleaseNotesGuide.rst`
```
Always start your release note with ``-`` so that it will be formatted as a bullet point in the collated list, e.g. `` - A new release note.``.
```

Reply:
I see, maybe we just need to remind people then if we see a failure on their PR due to mismatching `-` or `*` symbols. Is it ok if I harmonize all our release notes to use `-` in a separate PR so that this can go through?

Reply:
yep, that's a good idea. The problem is that usually that error says something like "bullet point list ended without a new line" which can be confusing.